palvelutarjotin/settings.py

```diff
@@ -124,8 +126,10 @@
 
 try:
     REVISION = subprocess.check_output(["git", "rev-parse", "--short", "HEAD"]).strip()
+    COMMITHASH = subprocess.check_output(["git", "rev-parse", "HEAD"]).strip()
 except Exception:
     REVISION = b"n/a"
+    COMMITHASH = ""
```

Comment:
I think it'd be better for maintenance to have only one variable here instead of two variables. REVISION is only used for [sentry_sdk init's release value](https://github.com/City-of-Helsinki/palvelutarjotin/blob/8214145bf65407f3e814af8415d58b79236a4293/palvelutarjotin/settings.py#L132) and in [django admin's site title](https://github.com/City-of-Helsinki/palvelutarjotin/blob/8214145bf65407f3e814af8415d58b79236a4293/palvelutarjotin/urls.py#L12) through [get_api_version](https://github.com/City-of-Helsinki/palvelutarjotin/blob/8214145bf65407f3e814af8415d58b79236a4293/common/utils.py#L49). Sentry sdk's release version [should be fine to change to commit long hash](https://docs.sentry.io/platforms/python/configuration/releases/#bind-the-version) according to their documentation:
> Commit SHA: If you use a version control system like Git, we recommend using the identifying hash
> (for example, the commit SHA, da39a3ee5e6b4b0d3255bfef95601890afd80709). 

and the django admin's site title getting longer is mostly a visual thing IMHO.

Reply:
fixed on #323